SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Reporting a Vulnerability
+If you think you have found a security vulnerability, please send a report to amundsen-security@lists.lfaidata.foundation. Please do not post security vulnerabilities on Slack.
+
+We don't currently have a PGP key, unfortunately.
+
+An Amundsen committer will send you a response indicating the next steps in handling your report. After the initial reply to your report, the committer will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
+
+*Important:* Please don't disclose the vulnerability before it have been fixed and announced, to protect our users.
+
+## Security announcements
+
+Please subscribe to the main mailing list, where we post notifications and remediation details for security vulnerabilities.
```

Comment:
Would be useful to add a link to that mailing list, @markgrover do you have it handy?

Reply:
there is an email link in the above (amundsen-security@lists.lfaidata.foundation). I ask the question in the channel as well, but do we know LF have any established handling security issue? or it relies on each project to handle on its own.

Reply:
Pasting from Slack, so it's visible publicly:
```
LF recommends:
Identify who on the project will be handling security issues (could be a team) 
Set up a separate email account to receive reporting (publish that on GH and project web site) 
Consider having the project become a CNA (CVE Numbering Authority)
```

Reply:
thanks

Reply:
Then this is not a "main" mailing list by the "security" mailing list, right?

Reply:
it's this https://lists.lfai.foundation/g/amundsen-announce , adding it to the PR